src/scss/base/_mixin.scss

```diff
@@ -0,0 +1,15 @@
+@mixin telefono {
+    @media (min-width:$telefono) {
+        @content;
+    }
+}
+@mixin tablet {
+    @media (min-width:$tablet) {
```

Comment:
can you add space between the ":" and "$tablet", to have something like "min-width: $tablet"